hasher-matcher-actioner/hmalib/common/tests/config_test.py

```diff
@@ -0,0 +1,152 @@
+# Copyright (c) Facebook, Inc. and its affiliates. All Rights Reserved
+
+from dataclasses import dataclass, field
+import unittest
+from unittest.mock import patch
+import os
+import typing as t
+
+import boto3
+import moto
+from moto import mock_dynamodb2
+
+from hmalib.common import config
+
+
+class ConfigTest(unittest.TestCase):
+
+    TABLE_NAME = "test-HMAConfig"
+
+    @staticmethod
+    def mock_aws_credentials():
+        """Mocked AWS Credentials for moto"""
+        os.environ["AWS_ACCESS_KEY_ID"] = "testing"
+        os.environ["AWS_SECRET_ACCESS_KEY"] = "testing"
+        os.environ["AWS_SECURITY_TOKEN"] = "testing"
+        os.environ["AWS_SESSION_TOKEN"] = "testing"
```

Comment:
Would it make sense to pass along the region here as AWS_DEFAULT_REGION? Rather than adding it to all resource getters?

Reply:
Yea, this is what I ended up doing to fix